katas/multiples/maria/README.md

```diff
@@ -0,0 +1,2 @@
+# Multiples of 3 or 5
```

Comment:
Same for this, please delete this file since the README file should not need to be duplicated